app/lib/dartdoc/dartdoc_runner.dart

```diff
@@ -382,3 +386,13 @@ class DartdocJobProcessor extends JobProcessor {
     _appendLog(logFileOutput, pr);
   }
 }
+
+bool _isKnownFailurePattern(String output) {
+  if (output.contains('Unhandled exception:') &&
+      output.contains('encountered ') &&
+      output.contains(' analysis errors') &&
+      output.contains('Dartdoc.logAnalysisErrors')) {
+    return true;
```

Comment:
Can we tighten these patterns? I imagine that legitimate infrastructure issues might well also produce these errors.

Reply:
Ah I see the &&. Maybe not so bad.

Reply:
`analysis errors` is coming from the exception and `Dartdoc.logAnalysisErrors` is coming from the top of the stack trace. I think this is a tight pattern for filtering out this specific issue.